### PR TITLE
fix: add possibility to include scrollable table + example in docs

### PR DESCRIPTION
**Pull request by JKMarkowski**
…n docs

#### Please provide a link to the associated issue.
https://github.com/SAP/fundamental-ngx/issues/704
#### Please provide a brief summary of this pull request.
I added directive which makes scrolling table horizontally possible.
#### If this is a new feature, have you updated the documentation?
Yes, the documentation is updated and contains one of usage example.

### CODE REVIEW

File: docs/app/documentation/component-docs/table/table-docs.component.ts [hunk 1]
```diff
@@ -8,10 +8,17 @@ import * as tableCheckHtml from '!raw-loader!./examples/table-checkboxes-example
...
+import * as tableResponsiveHtml from '!raw-loader!./examples/table-responsive-example.component.html';
+import * as tableResponsiveTs from '!raw-loader!./examples/table-responsive-example.component.ts';
 
 @Component({
     selector: 'app-table',
-    templateUrl: './table-docs.component.html'
+    templateUrl: './table-docs.component.html',
+    styles: [`
+        ::ng-deep app-table .fd-tile {
+            display: block;
```
Comment (MattL75) [inline docs/app/documentation/component-docs/table/table-docs.component.ts]: hmm kind of annoying that this is needed in the first place, but I get it and it's fine to keep it. Maybe something we can consider improving later

**Comment by mikerodonnell89** [comment]
This works really good, but I am wondering if we want to test for div width and apply the fixed class to the parent based on that rather than have it as a directive?  Could use the scrollbar width changes for frame of reference but this might be something we should check with the fundamental design team

**Comment by JKMarkowski** [comment]
Actually in fiori fundamental library there is no such thing allowing to scroll horizontally. --fixed class is more useful on adding fixed cell, so it's not needed to be used in this case.

**Comment by MattL75** [comment]
Yeah I'm good to merge this - maybe we could add the fixed cell at some point, or even make it toggle-able in the example. I'll wait for word from @mikerodonnell89 though.

**Pull request merged**